content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -39,35 +39,47 @@ certain amount of money. They can be used to provide a full refund to a customer
 remaining amount due. While they are usually created through the *Accounting* or *Invoicing*
 applications, they can be created through a *Helpdesk* ticket, as well.
 
-.. note::
+.. important::
    Invoices must be posted before a credit note can be generated.
 
-To create a credit note, navigate to a ticket on the :menuselection:`Helpdesk` application, and
-click the :guilabel:`Refund` button in the upper-left corner of the ticket dashboard. Then, select
-the corresponding invoice from the :guilabel:`Invoices to Refund` drop-down menu.
+To create a credit note, navigate to a ticket on the :menuselection:`Helpdesk app`, and click the
+:guilabel:`Refund` button in the top-left corner of the ticket dashboard. This opens a
+:guilabel:`Refund` pop-up window.
```

Comment:
```suggestion
:guilabel:`Refund` button in the top-left corner of the ticket form. This opens a
:guilabel:`Refund` pop-up window.
```

not sure if form is the right word, but the refund button is on the individual ticket instead of the dashboard.